app/views/warnings/_scheme_12_fees_summary.html.haml

```diff
@@ -0,0 +1,8 @@
+- if claim&.last_injection_attempt&.succeeded && claim.has_scheme_12_fees?
+  .js-callout-scheme-12_fees-warning
```

Comment:
what does this do, and name should be `js-callout-scheme-12-fees-scheme` by the looks of it

Reply:
I believe it just adds the class js-callout.. to the warning message box. I have amended it to .js-callout-injection-warning.

Reply:
do you know the impact of this @naseberry